<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Display waveform</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QCustomPlot" name="customPlot_two" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QCustomPlot" name="fft_two" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QCustomPlot" name="fft_one" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QCustomPlot" name="customPlot_one" native="true"/>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QCustomPlot" name="axisOrbit" native="true">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4" columnminimumwidth="1,0">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Serial Port</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>StopBits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BaudRate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>DataBits</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="baudRate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="currentText">
             <string>115200</string>
            </property>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="portSelect">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="maxVisibleItems">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>PortName</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="stopBits">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="openPort">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>openPort</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="dataBits">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Operation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Parity</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="parity">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Even</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Odd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Space</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mark</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Analyses</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="connect_db">
            <property name="text">
             <string>ConnDB</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="start_collect">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="freq">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="freq_num">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="hanning">
            <property name="text">
             <string>Hanning</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="hamming">
            <property name="text">
             <string>Hamming</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="export_data">
            <property name="text">
             <string>ExportDatas</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="clear_db">
            <property name="text">
             <string>ClearDatas</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
